data/json/npcs/isherwood_farm/NPC_Jack_Isherwood.json

```diff
@@ -0,0 +1,356 @@
+[
+  {
+    "type": "npc",
+    "id": "isherwood_jack",
+    "//": "Patriarch of the Isherwood family.",
+    "name_unique": "Jack Isherwood",
+    "gender": "male",
+    "class": "NC_ISHERWOOD_JACK",
+    "attitude": 0,
+    "mission": 7,
+    "chat": "TALK_ISHERWOOD_JACK",
+    "mission_offered": "MISSION_ISHERWOOD_JACK_1",
+    "faction": "isherwood_family"
+  },
+  {
+    "type": "npc_class",
+    "id": "NC_ISHERWOOD_JACK",
+    "name": "Jack Isherwood",
+    "job_description": "Im the patriarch of the Isherwood family",
+    "traits": [ { "group": "NPC_starting_traits" }, { "group": "Appearance_demographics" } ],
+    "common": false,
+    "bonus_per": { "one_in": 4 },
+    "shopkeeper_item_group": "NC_ISHERWOOD_JACK_misc",
+    "skills": [
+      {
+        "skill": "ALL",
+        "level": { "mul": [ { "one_in": 3 }, { "sum": [ { "dice": [ 2, 2 ] }, { "constant": -2 }, { "one_in": 4 } ] } ] }
+      },
+      { "skill": "barter", "bonus": { "rng": [ 2, 4 ] } },
+      { "skill": "gun", "bonus": { "rng": [ 4, 8 ] } },
+      { "skill": "melee", "bonus": { "rng": [ 3, 6 ] } },
+      { "skill": "survival", "bonus": { "rng": [ 3, 6 ] } },
+      { "skill": "pistol", "bonus": { "rng": [ 3, 6 ] } },
+      { "skill": "rifle", "bonus": { "rng": [ 4, 8 ] } }
+    ]
+  },
+  {
+    "type": "item_group",
+    "id": "NC_ISHERWOOD_JACK_shirt",
+    "items": [ [ "tshirt", 20 ], [ "dress_shirt", 20 ] ]
+  },
+  {
+    "type": "item_group",
+    "id": "NC_ISHERWOOD_JACK_gloves",
+    "items": [ [ "null", 60 ], [ "gloves_leather", 20 ], [ "gloves_fingerless", 20 ] ]
+  },
```

Comment:
There are item groups above and below that are completely identical to the Claire's item groups. There's no need for duplicate groups, you can use one item group for both Isherwoods.

Reply:
didn't know about this, thanks.